src/Vex/CommandContext.cpp

```diff
@@ -709,7 +734,8 @@ std::optional<SyncToken> CommandContext::Submit()
     hasSubmitted = true;
 
     std::vector<SyncToken> tokens = backend->EndCommandContext(*this);
-    return !tokens.empty() ? std::optional{ tokens[0] } : std::nullopt;
+    VEX_ASSERT(!tokens.empty());
```

Comment:
```suggestion
    VEX_ASSERT(tokens.size() == 1);
```
We probably want to ensure that any additional tokens are not ignored (there should never be more than 1).